jaeger-core/src/main/java/io/jaegertracing/propagation/B3TextMapCodec.java

```diff
@@ -14,9 +14,9 @@
 
 package io.jaegertracing.propagation;
 
-import io.jaegertracing.SpanContext;
-import io.jaegertracing.Tracer;
-import io.jaegertracing.baggage.BaggageRestrictionManager;
+import io.jaegertracing.JaegerSpanContext;
+import io.jaegertracing.JaegerTracer;
+import io.opentracing.SpanContext;
```

Comment:
this is a semantic change, previously methods were accepting Jaeger context

Reply:
Reverted.